bosh/opsfiles/clients.yml

```
@@ -144,6 +144,17 @@
     redirect-uri: https://dashboard.((system_domain))/pp/v1/auth/sso_login_callback
     secret: ((stratos-client-secret))
 
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaa/clients/cg-ui?
+  value:
+    override: true
+    authorized-grant-types: authorization_code,client_credentials,refresh_token
+    autoapprove: true
+    secret: ((cg-ui-client-secret))
+    scope: cloud_controller.read,cloud_controller.admin_read_only,cloud_controller.global_auditor,cloud_controller.write,scim.me,openid,profile,uaa.user,cloud_controller.admin
```

```suggestion
    scope: scim.me,openid,profile,uaa.user,cloud_controller.admin
```

If you have `cloud_controller.admin` you have everything else, so we can simplify this a lot

https://docs.cloudfoundry.org/concepts/architecture/uaa.html#cc-scopes